### PR TITLE
📊 cancer: IHME share of cancer deaths by type

### PR DESCRIPTION
**Pull request by veronikasamborska1994**
hi @spoonerf ! sorry for another review request but since you normally do the IHME updates I figured you are the best person to ask for this :) let me know if you are too busy with other things though and I'll ask someone else!

### CODE REVIEW

**Comment by veronikasamborska1994** [comment]
closed as didn't see any particularly interesting patterns in the data for now

**Comment by spoonerf** [comment]
@veronikasamborska1994 no worries at all, happy to review :)

File: etl/steps/data/garden/ihme_gbd/2024-09-16/gbd_cancers_deaths.py [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+        & (tb["country"] == "World")
+        & (tb["year"] == tb["year"].max())
+    ]
+
+    # Identify cancers to aggregate
+    cancers_to_aggregate = cancers_tb[cancers_tb["value"] < 200000]["cause"].drop_duplicates().tolist()
+    cancers_to_aggregate = cancers_to_aggregate + other_cancers
+
+    # Log the cancers that were grouped]
+    paths.log.info(f"Cancers grouped into 'Other cancers (OWID)': {', '.join(cancers_to_aggregate)}")
```
Comment (spoonerf) [inline etl/steps/data/garden/ihme_gbd/2024-09-16/gbd_cancers_deaths.py:80]: Nice

**Review by spoonerf — APPROVED** [review]
LGTM!

**Pull request merged**